frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -100,6 +100,7 @@ interface ITableContainerProps<T = any> {
   setExportRows?: (rows: Row[]) => void;
   resetPageIndex?: boolean;
   disableTableHeader?: boolean;
+  show0Count?: boolean;
```

Comment:
TODO, clean this up

Reply:
https://fleetdm.slack.com/archives/C01EZVBHFHU/p1714083808183799?thread_ts=1713887183.358959&cid=C01EZVBHFHU